src/components/app/MenuButtons.css

```diff
@@ -167,6 +167,13 @@
   white-space: nowrap;
 }
 
+.photon-input.menuButtonsPermalinkTextField {
+  /* the chain class selectors are used here 
+  to override the styling for the photon-input class */
+  height: 13px;
+  padding: 3px;
+}
```

Comment:
Actually, the same here; I think it's better without this override. What do you think?